core/domain/services/graphql/types/Attendee.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+namespace EventEspresso\core\domain\services\graphql\types;
+
+use EEM_Attendee;
+use EE_Attendee;
+use EventEspresso\core\services\graphql\fields\GraphQLFieldInterface;
+use EventEspresso\core\services\graphql\types\TypeBase;
+use EventEspresso\core\services\graphql\fields\GraphQLField;
+use EventEspresso\core\services\graphql\fields\GraphQLOutputField;
+use WPGraphQL\AppContext;
+use GraphQL\Type\Definition\ResolveInfo;
+
+/**
+ * Class Attendee
+ * Description
+ *
+ * @package EventEspresso\core\domain\services\graphql\types
+ * @author  Brent Christensen
+ * @since   $VID:$
+ */
+class Attendee extends TypeBase
+{
+
+    /**
+     * Attendee constructor.
+     *
+     * @param EEM_Attendee $attendee_model
+     */
+    public function __construct(EEM_Attendee $attendee_model)
+    {
+        $this->model = $attendee_model;
+        $this->setName($this->namespace . 'Attendee');
+        $this->setIsCustomPostType(false); // Set to false to use our model queries
+        parent::__construct();
+    }
+
+
+    /**
+     * @return GraphQLFieldInterface[]
+     * @since $VID:$
+     */
+    public function getFields()
+    {
+        return [
+            new GraphQLField(
+                'id',
+                ['non_null' => 'ID'],
+                null,
+                esc_html__('The globally unique ID for the object.', 'event_espresso')
+            ),
+            new GraphQLOutputField(
+                'dbId',
+                ['non_null' => 'Int'],
+                'ID',
+                esc_html__('The attendee ID.', 'event_espresso')
+            ),
+            new GraphQLOutputField(
+                'cacheId',
+                ['non_null' => 'String'],
+                null,
+                esc_html__('The cache ID of the object.', 'event_espresso')
+            ),
+            new GraphQLOutputField(
+                'address',
+                'String',
+                'address',
+                esc_html__('Address Part 1', 'event_espresso'),
+                null,
+                null,
+                ['ee_edit_contacts']
+            ),
+            new GraphQLOutputField(
+                'avatar',
+                'String',
+                null,
+                esc_html__('Address Part 1', 'event_espresso'),
+                null,
+                [$this, 'getAvatar'],
+                ['ee_edit_contacts']
+            ),
+            new GraphQLOutputField(
+                'address2',
+                'String',
+                'address2',
+                esc_html__('Address Part 2', 'event_espresso'),
+                null,
+                null,
+                ['ee_edit_contacts']
+            ),
```

Comment:
that middle output field is a mix of avatar and address 1
maybe fix its description then place it after the address 2 field

Reply:
Oops! 🤦‍♂️